app/package.json

```diff
@@ -135,7 +135,7 @@
     "glob": "~7.1.6",
     "mini-css-extract-plugin": "~0.9.0",
     "npm-run-all": "^4.1.5",
-    "playwright": "^1.12.3",
+    "playwright": "^1.17.1",
```

Comment:
Looks like this PR might be bringing some additional changes?

Reply:
oh i think i messed something up merging. ill have a look over the weekend. ill fix this soon.